src/about-menu/open-about-menu.ts

```diff
@@ -39,7 +40,14 @@ export function openAboutWindow() {
 		if ( aboutWindow ) {
 			// Inject version into the about window's HTML
 			aboutWindow.webContents
-				.executeJavaScript( `document.getElementById('version').innerText = '${ packageJson }'` )
+				.executeJavaScript(
+					`document.getElementById('version').innerText = '${ packageJson }'; 
+					document.getElementById('studio-by-wpcom').innerText = '${ __( 'Studio by WordPress.com' ) }';
+					document.getElementById('version-text').innerText = '${ __( 'Version' ) }'; 
```

Comment:
Let's unify the word `Version` and the actual number. Something like `sprintf( __( 'Version %s' ), packageJson )`

Reply:
Added in https://github.com/Automattic/studio/pull/228/commits/af02dbb33cd562e5aa90cddf51fc6864f72e0ace